hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -156,6 +170,11 @@ public void initialize(MerkleInternal previous) {
 			log.info("Created disk file system after <=0.9.0 state restoration");
 			skipDiskFsHashCheck = true;
 		}
+		if (runningHashLeaf() == null) {
+			setChild(ChildIndices.RECORD_STREAM_RUNNING_HASH,
+					new RecordsRunningHashLeaf(new RunningHash()));
+			log.info("Created RecordsRunningHashLeaf after <=0.110.0 state restoration");
```

Comment:
nit : Should be`0.11.0` instead of `0.110.0`

Reply:
modified. Thanks @Neeharika-Sompalli !